inference-engine/ie_bridges/python/src/openvino/offline_transformations/offline_transformations_api_impl.cpp

```diff
@@ -28,3 +31,20 @@ void InferenceEnginePython::ApplyLowLatencyTransformation(InferenceEnginePython:
     });
     manager.run_passes(network.actual->getFunction());
 }
+
+void InferenceEnginePython::CheckAPI() {
```

Comment:
What is the purpose of this method?
In release mode asserts do nothing. Is it expected?

Reply:
The purpose is just to call some ngraph functionally but not just check imports. This pass is temporary and I future will be replaced with regular ngraph transformations.

Reply:
I propose to add a comment. In order to understand what happens. It can be done in the separate PR.